apis/keda/v1alpha1/scaledobject_webhook.go

```diff
@@ -149,10 +149,15 @@ func verifyHpas(incomingSo *ScaledObject, action string) error {
 			}
 
 			if !owned {
-				err = fmt.Errorf("the workload '%s' of type '%s' is already managed by the hpa '%s'", incomingSo.Spec.ScaleTargetRef.Name, incomingSoGckr.GVKString(), hpa.Name)
-				scaledobjectlog.Error(err, "validation error")
-				prommetrics.RecordScaledObjectValidatingErrors(incomingSo.Namespace, action, "other-hpa")
-				return err
+				if incomingSo.ObjectMeta.Annotations["keda.sh/transfer-hpa-ownership"] == "true" &&
```

Comment:
Could you please define the annotation here? https://github.com/kedacore/keda/blob/main/apis/keda/v1alpha1/scaledobject_types.go#L50